platform/core/src/services/ToolBarService/ToolbarService.ts

```diff
@@ -181,6 +179,18 @@ export default class ToolbarService extends PubSubService {
     return [this.state.primaryToolId, ...Object.keys(this.state.toggles)];
   }
 
+  public setActive(id: string, isActive: boolean) {
+    if (isActive) {
+      this.state.toggles[id] = true;
+    } else {
+      delete this.state.toggles[id];
+    }
+  }
```

Comment:
I had to read the implementation that it means `setToggleActive`, how about a rename?

Reply:
I vote for a name change here too.